contributors.js

```diff
@@ -1,8 +1,8 @@
 [
     {
-        name: "Micha",
-        linkedIn: "https://linkedin.com/in/michaella-rodriguez",
-        gitHub: "https://github.com/michaella23",
+        name: "Erik",
```

Comment:
Hey @eram23, you will need to put your info inside of a separate object, after mine. Kind of like this -
```[
       { (me)
        name: "",
        linkedIn: "",
        gitHub: "",
        avatar: ""
      },
      { (you)
             name: "",
       linkedIn: "",
       gitHub: "",
       avatar: ""
       },
     ]

Reply:
Understood, will update it now thanks.